src/styles/main.styl

```diff
@@ -5,7 +5,11 @@ html, body
     width: 0
     height: 0
     clear: both
-
+textarea {
```

Comment:
No `{`
Should override the user-agent stylesheet for font-family, so it is not monospace, too.